bin/test_schema

```diff
@@ -31,7 +31,7 @@ if [ "$build" == y ]; then
     validator/bin/build
 fi
 
-jarfile=$(realpath validator/build/libs/validator-1.0-SNAPSHOT-all.jar)
+jarfile="$(cd "$(dirname "validator/build/libs/validator-1.0-SNAPSHOT-all.jar")"; pwd)/$(basename "validator/build/libs/validator-1.0-SNAPSHOT-all.jar")"
```

Comment:
should split out the root-path thing into a separate variable... too ugly to comprehend otherwise.  Also, 'basename' and 'dirname' not necessary with a static string -- you can just use the filename directly.

Reply:
I've put the JAR filename into a different variable and removed the static string, I think it's easier to read now.